src/utils/uploads.js

```diff
@@ -335,10 +337,16 @@ function getDataTransferItemId(
 }
 
 /**
- * Multiput uploads require the use of crypto, which is only supported in secure contexts
+ * Multiput uploads require the use of crypto, which is only supported in secure contexts.
+ * Multiput uploads is not supported on mobile iOS Safari.
```

Comment:
nit: I am wondering if we should add any more details here, around why we took the decision to not support it ?